lib/tasks/data_hygiene.rake

```diff
@@ -52,4 +52,36 @@ namespace :data_hygiene do
       speech.save(validate: false)
     end
   end
+
+  desc "Make attachments' command paper numbers valid"
+  namespace :make_command_paper_numbers_valid do
+    def call_attachment_attribute_updater(dry_run:)
+      Attachment
+        .where.not(command_paper_number: nil)
+        .where.not(command_paper_number: "")
+        .each do |attachment|
+        old_attr = attachment.command_paper_number
+        begin
+          new_attr = DataHygiene::AttachmentAttributeUpdater.call(attachment, dry_run: dry_run)
+          if old_attr == new_attr
```

Comment:
This isn't quite right, since the updater will have done an `update` already, unless you're relying on [ActiveRecord::Dirty](https://api.rubyonrails.org/classes/ActiveModel/Dirty.html)?

Reply:
Ah yes, good spot. At one point this was true, when I was handling the `update` call in the rake task itself. I'll fix it 👍